### DISCUSSION TITLE
Predisposizione centralino

### DISCUSSION THREAD

**Discussion by fmulassano** [indomus/forum#1823] [GENERALE - Discussioni generiche]
bonjour!
allora, forse riesco a separare in parte l'impianto di casa nuova facendo delle tracce e nuovi tubi dal centralino fino ad una cassetta di derivazione da cui si dirama tutto l'appartamento ma alla quale ora arriva solo 1 linea dall'unico magnetotermico diff che c'è nel micro quadretto.

quindi pensavo di provare a fare

MT-D 4 moduli
Shelly 4PRO pm 3 moduli (*)

_Magnetotermici tutti da 1M_ quindi seziono solo la fase
Lavatrice (*)
Lavastoviglie (*)
Forno (*)
Resto della cucina (*)

Caldaia 
Sala
Camera
Studio

Presa di servizio nel contatore (magari metto il modem li, però ho pareti molto spesse, dovrei mettere un ripetitore su linea elettrica... oppure?)

Predisposizione per condizionatore (in futuro)
e sono 17

non vorrei mettere un 24... 
**cosa mi consigliate?**

_Sapendo che:_

- probabilmente camera e studio non riuscirò a separarli perchè se va bene c'è una linea sola che va in zona notte e i fili nei tubi non si muovono manco a morire
- Caldaia potrei raggrupparla a qualcos'altro
- Non riuscirò in sala / camera e studio a separare luce e forza

Lo Shelly 4PRO monitorerebbe 4 linee 
cosa potrei mettere se volessi monitorare anche le altre linee? 
avrebbe senso? Sarebbe utile? 

Il tutto poi finirebbe per essere controllato da HA con Alexa e telefono
(non ho ancora affrontato il discorso HA, me lo studierò questo inverno (avrò tempo da dopo novembre)
però ho un paio di raspberry 3/4, sono un informatico, pian piano penso di cavarmela.

Quello che mi preme ora è la predisposizione.

Ciao

**Comment by Nik71git** [comment]
se sei in un momento in cui puoi fare tracce ti consiglio di allargare la zona centralino e predisporre l'impianto per bene. 
secondo me la caldaia deve stare da sola e in sala devi riuscire a separare 10A da 16A, se hai un cavo solo di fase allora non c'è separazione e viaggia tutto a 10A o 16A. Gli Shelly vanno bene per monitorare ma sono ingombranti, per avere lo stesso risultato puoi mettere degli EM installati anche fuori dai binari DIN. Infine i magneto termici ad un modulo esistono anche 1P+N in modo da sezionare fase e neutro che è il minimo....
